### PR TITLE
More package metadata

### PR DESCRIPTION
**Pull request by Luthaf**
The root README needs to be markdown instead of rst since Cargo only supports markdown.

----
:books: Documentation preview :books:: https://metatensor--384.org.readthedocs.build/en/384/

### CODE REVIEW

**Comment by PicoCentauri** [comment]
> The root README needs to be markdown instead of rst since Cargo only supports markdown.

WHAAAAT!!!

File: README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Metatensor
```
Comment (PicoCentauri) [inline README.md:1]: Aren't importing this somewhere in our docs? If yes, this will be broken now. If no we are fine.

**Comment by agoscinski** [comment]
> Metatensor is a self-describing sparse tensor data format supporting gradients for atomistic machine learning and beyond.

For me the gradient support is an essential part that contrasts this package from others. But maybe the description now too long. Feel free to use your version also, if you feel like this not suitable. Don't want to overthink this one line description.

File: README.md (hunk 1 above)
Comment (Luthaf) [reply README.md:1]: Fairly sure we don't import the README, only Contributing

**Comment by Luthaf** [comment]
I could do something like "Metatensor is a self-describing sparse tensor data format for atomistic machine learning and beyond; storing values and gradients of these values together." in the README, and keep the shorter one for the different packages descriptions.

> > The root README needs to be markdown instead of rst since Cargo only supports markdown.
> 
> WHAAAAT!!!

Yep, rst is almost only used in the Python ecosystem. Everyone else is using markdown.

File: README.md (hunk 1 above)
Comment (PicoCentauri) [reply README.md:1]: okay then I am fine.

**Comment by PicoCentauri** [comment]
> I could do something like "Metatensor is a self-describing sparse tensor data format for atomistic machine learning and beyond; storing values and gradients of these values together." in the README, and keep the shorter one for the different packages descriptions.

Yes fine for me.

>>> The root README needs to be markdown instead of rst since Cargo only supports markdown.

>> WHAAAAT!!!

>Yep, rst is almost only used in the Python ecosystem. Everyone else is using markdown.

I know and it is fine for to change it.

**Review by PicoCentauri — APPROVED** [review]

**Pull request merged**